content/2015.07.cliquet.rationale.rst

```diff
@@ -0,0 +1,518 @@
+Pourquoi avons-nous fait Cliquet ?
+##################################
+
+:lang: fr
+:url: pourquoi-cliquet
+:slug: post-cliquet-rationale
+:date: 2015-07-14
+:summary: ...
+
+
+**tldr; Cliquet est un toolkit Python pour construire des APIs, qui implémente
```

Comment:
Maybe we should have the tl-dr read from the summary in the article rather than having to repeat it?